### PR TITLE
coverage for calculateDistributeExcessIdleShareProceedsNetLongEdgeCaseSafe

### PR DESCRIPTION
**Pull request by jrhea**
This PR should cover all branches for `calculateDistributeExcessIdleShareProceedsNetLongEdgeCaseSafe()`

### CODE REVIEW

File: contracts/test/MockLPMath.sol [hunk 1]
```diff
@@ -100,4 +100,14 @@ contract MockLPMath {
                 _bondAmount
             );
     }
+
+    function calculateDistributeExcessIdleShareProceedsNetLongEdgeCaseSafe(
```
Comment (mcclurejt) [inline contracts/test/MockLPMath.sol:104]: needs more verbosity

Comment (jrhea) [reply contracts/test/MockLPMath.sol:104]: function not long enough

**Comment by mcclurejt** [comment]
what's the lift on hitting [this branch](https://github.com/delvtech/hyperdrive/blob/code-coverage/contracts/src/libraries/LPMath.sol#L1070) as well?

**Review by mcclurejt — APPROVED** [review]
more coverage = good

**Comment by jrhea** [comment]
> what's the lift on hitting [this branch](https://github.com/delvtech/hyperdrive/blob/code-coverage/contracts/src/libraries/LPMath.sol#L1070) as well?

it is already covered:

Screenshot_20240412-013014

**Pull request merged**